l10n/templates/strings.properties

```diff
@@ -81,11 +99,12 @@ prefs_section_rows_option={num} row;{num} rows
 prefs_search_header=Web Search
 prefs_topsites_description=The sites you visit most
 prefs_topstories_description2=Great content from around the web, personalized for you
-# LOCALIZATION NOTE (prefs_topstories_show_sponsored_label): {provider} is
-# replaced by the name of the content provider for this section, e.g., "Pocket"
-prefs_topstories_show_sponsored_label={provider} Sponsored Stories
+prefs_topstories_options_sponsored_label=Sponsored Stories
 prefs_topstories_sponsored_learn_more=Learn more
 prefs_highlights_description=A selection of sites that you’ve saved or visited
+prefs_highlights_options_visited_label=Visited Pages
+prefs_highlights_options_download_label=Most Recent Download
+prefs_highlights_options_pocket_label=Pocket Saves
```

Comment:
This is the only string I'm not sure about. Is "Saves" something that shows up in Pocket's UI, or just a generic short term for "saved items"? Are there space limitations? I would assume from the image that the text can wrap.

Reply:
Sorry, I meant their website UI. The Pocket add-on is, to put it mildly, unmaintained.

And I realized I could just check their website 🐢 It doesn't look like there's anything similar to "Saves" in the UI.